src/App.tsx

```diff
@@ -1,25 +1,47 @@
-import React from 'react';
-import logo from './logo.svg';
+import axios from 'axios';
+import { useEffect, useState } from 'react';
 import './App.css';
 
+export interface Breed{
+  name: string
+}
+export interface GetBreedsResponse {
+  data: Breed[]
+}
+
+export const getBreeds = async (): Promise<Breed[]|any>  => {
+  try{
+    return await axios.get('https://api.thecatapi.com/v1/breeds');
+  }catch(error){
+    return error;
+  }
+}
+
 function App() {
+  
+
+  
+  const [cats,setCats] = useState<Breed[]>([]);
+
+useEffect(()=>{ 
+  // bora chamar a API
+  getBreeds()
+    .then(({data}) => {
+      setCats(data)
+  })
+
+},[])
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.tsx</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div>
+
+     Hello World
+     <ul>
+    {cats.map((item)=>(
+     <li key={`${item.name}`}>{item.name}</li>
+    ))}
+    </ul>
```

Comment:
Vamos começar a estruturar melhor o código e permitir que a aplicação cresça de forma escalável? Podemos criar um componente que vai ter a list e essa parte do código movida pra lá. 
```suggestion
     <Cats />
```

Seria legal já pensar em uma estrtura de pastas para a sua aplicação.